### PR TITLE
chore: add documentation to new docs folder

### PR DESCRIPTION
**Pull request by awentzel**
# Description
This migrates over previous published documentation to new folder structure.  This does not modify the old structure. I presume one we have all the documentation we want from the old root docs folder we can add a cleanup task to remove on one PR.

## Motivation & context
Content updates

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/website/src/docs/community/build-and-test.md [hunk 1]
```diff
@@ -3,4 +3,97 @@ id: build-and-test
...
+
+* Start a web server on a local port (eg, [http://localhost:8000](http://localhost:8000)) and use the command line interface to request a tunnel to the local server:
+
+```bash
+lt --port 8000
+```
+
+* You receive a URL to share as long as your local instance remains active. Requests route to your local service at the specified port.
+
+### Testing with Sauce Labs
```
Comment (nicholasrice) [inline sites/website/src/docs/community/build-and-test.md:64]: Does this actually work? Do we have a sauce-labs account?

Comment (awentzel) [reply sites/website/src/docs/community/build-and-test.md:64]: Yeah.  Additionally, I just logged in with my GitHub account and looks like Screener.io is now part of Sauce Labs.  I know quite a few internal teams use Screener.io https://screener.io/

Comment (awentzel) [reply sites/website/src/docs/community/build-and-test.md:64]: When people are ready to use this, simply send me a request and I can invite you to our organization account.  The comments in code explain how to configure user/key as env variables to integration and run the script.

**Comment by awentzel** [comment]
@chrisdholt I didn't want to delete old docs quite yet just in case Rob wanted to extract any other information of use.  We seem to have three locations now `./docs` `./sites/website/docs`, and `./sites/website/src/docs`  @EisenbergEffect am I updated the correct files?  Would you like me to delete the original files or perform a single cleanup of docs when we are comfortable with the new set of docs?  Thanks

File: sites/website/src/docs/community/build-and-test.md (hunk 1 above)
Comment (nicholasrice) [reply sites/website/src/docs/community/build-and-test.md:64]: Shouldn't that be part of this content?

Comment (nicholasrice) [reply sites/website/src/docs/community/build-and-test.md:64]: This feels like a disconnect if we're telling people to build and test using these steps but we first have to add them to our organization. Will we add *anyone* to that organization? Are these steps really intended for the community?

Sorry if I'm just missing something here.

Comment (awentzel) [reply sites/website/src/docs/community/build-and-test.md]: I can see that perspective.  I've moved the Sauce Labs docs off of the site docs and into `./build/testing` with the rest of the SL details.

**Comment by EisenbergEffect** [comment]
@awentzel If you can merge this, I can take it from here and make the rest of the docs fixes this weekend.

**Pull request merged**